Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -121,6 +121,19 @@ final class Analytics {
 
         tracker.track(event)
     }
+    
+    func userSearchViaBingABTest() {
+        let event = Structured(category: Category.abTest.rawValue,
+                              action: "user_search")
+            .label("search_key")
+
+        // add A/B Test context
+        if let context = Self.getTestContext(from: .bingSearch) {
```

Comment:
That's indeed the expected behavior 👍 
I left the code similar to the other experiment in case we'll add some variants.
What we need at the moment is to send `category`, `label` and `action`. Which the function does 👍 

```
(lldb) po event.category
"ab_Test"

(lldb) po event.action
"user_search"

(lldb) po event.label
▿ Optional<String>
  - some : "search_key"
```